content/graphqlmoredata/3.md

```diff
@@ -0,0 +1,32 @@
++++
+date = "2020-09-15T00:00:00+00:00"
+next = "/?graphql"
+prev = "/graphqlmoredata/3"
```

Comment:
This step is not showing up. By clicking we are redirected before the page appears.

Reply:
Fixed. This 3rd lesson was added after moving the module. The 2nd lesson now links to the 3rd and the 3rd links back to the 2nd. https://deploy-preview-351--dgraph-tour.netlify.app/tour/graphqlmoredata/3/